sidebars.js

```diff
@@ -293,6 +293,13 @@ module.exports = {
         { type: 'doc', id: 'deployment_patterns/index', label: 'Index' }
       ]
     },
+    {
+      type: 'category',
+      label: 'Integrations',
+      items:[
+        { type: 'doc', id: 'integrations/index', label: 'Index' }
+      ]
```

Comment:
```suggestion
      items:[
        {type: 'doc', id: 'integrations/integration_datahub'},
        { type: 'doc', id: 'integrations/index', label: 'Index' }
      ]
```

This should display your integration doc as an item in the table of contents itself, right above the index page for the integrations category.  I apologize for not suggesting these changes in the last pass: I can only admit that I was more focused on the contents of `integration_datahub.md` than the changes to the table of contents.